astropy/table/soco.py

```diff
@@ -7,12 +7,10 @@
 
 from collections import OrderedDict
 from itertools import starmap
+from astropy.utils.compat.optional_deps import HAS_SORTEDCONTAINERS
 
-try:
+if HAS_SORTEDCONTAINERS:
```

Comment:
Will renaming `HAS_SOCO` to `HAS_SORTEDCONTAINERS` break anything downstream? Maybe @taldcroft knows.

Reply:
I don't think these things were ever exposed as public API... Also don't see any reference to this in docs. Also don't see any uses of this in other github projects https://github.com/search?p=1&q=HAS_SOCO&type=Code